plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions.php

```diff
@@ -78,11 +78,27 @@ protected function render( $attributes, $content, $block ) {
 
 		$product_type = $product->get_type();
 
+		$slug = $product_type . '-product-add-to-cart-with-options';
+
 		if ( in_array( $product_type, array( ProductType::SIMPLE, ProductType::EXTERNAL, ProductType::VARIABLE, ProductType::GROUPED ), true ) ) {
+			$template_part_path = Package::get_path() . 'templates/' . BlockTemplateUtils::DIRECTORY_NAMES['TEMPLATE_PARTS'] . '/' . $slug . '.html';
+		} else {
+			/**
+			 * Filter to declare product type's cart block template is supported.
+			 *
+			 * @since 9.9.0
+			 * @param mixed string|boolean The template part path if it exists
+			 * @param string $product_type The product type
+			 */
+			$template_part_path = apply_filters( '__experimental_woocommerce_' . $product_type . '_add_to_cart_with_options_block_template_part', false, $product_type );
+		}
+
+		if ( is_string( $template_part_path) && file_exists( $template_part_path ) ) {
```

Comment:
@Aljullu This is the lint error 

>96 | ERROR | Expected 1 spaces before closing parenthesis; 0 found (PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket)

I don't understand it though... it's saying there's no space before the closing `)` ? when there definitely looks to be one.

Reply:
NM... I see the missing space is in the `is_string()` check